services/dockerhub.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: page
+---
+
+# [Docker Hub](https://docs.docker.com/docker-hub/)
+
+Docker Hub is a registry service that builds and stores Docker images[1](#notes) providing a centralized resource for image distribution and change management.
+
+We refer to the [Docker documentation](https://docs.docker.com/docker-hub/builds/) to integrate GitHub with Docker Hub.
+
+Each GitHub repository is connected to Docker Hub through an `Automated build` using the following build settings.
+
+| Type 	| Name 	| Dockerfile Location 	| Docker Tag Name 	|
+|------	|------	|---------------------	|-----------------	|
+| Tag  	| /.*/ 	| /                   	| latest          	|
+| Tag  	| /.*/ 	| /                   	| Same as tag     	|
```

Comment:
Are we still using version X.Y.Z when drafting the release through inforbot?

Reply:
No.
Milestones are simply named as YYYY.X where YYYY is the year when the milestone is opened and X is a progressive number.

it drafts, from the `master` branch, the MILESTONE release ...

Releases will be names as milestones.

Reply:
Ok, thank you,